src/components/notification/__tests__/NotificationsWrapper.test.js

```diff
@@ -7,7 +7,25 @@ describe('components/notification/NotificationsWrapper', () => {
     test('should render a Portal with the notifications-wrapper class', () => {
         const wrapper = shallow(<NotificationsWrapper />);
         expect(wrapper.is('Portal')).toBeTruthy();
+        const portal = wrapper.find('Portal');
         expect(wrapper.hasClass('notifications-wrapper')).toBeTruthy();
+        expect(portal.props('aria-live')).toBeTruthy();
+    });
+
+    test('should render a focus trap', () => {
+        const wrapper = shallow(
+            <NotificationsWrapper>
+                <Notification>test1</Notification>
+            </NotificationsWrapper>,
+        );
+        const focusTrap = wrapper.find('FocusTrap');
+        expect(focusTrap.length).toEqual(1);
+    });
+
+    test('should not render focusTrap if there are no children', () => {
+        const wrapper = shallow(<NotificationsWrapper />);
+        const focusTrap = wrapper.find('FocusTrap');
+        expect(focusTrap.length).toEqual(0);
```

Comment:
You can also use `expect(wrapper.exists('FocusTrap')).toBe(false);`, if you prefer. It was improved in a recent version of Enzyme.